tests/e2e-container/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM quay.io/ukhomeofficedigital/systemd
+
+RUN dnf upgrade -y && \
+    dnf install \
+    dnsmasq \
+    jq \
+    which \
+    dnf-plugins-core \
+    -y && \
+    dnf clean all
+
+# Add Docker in Docker...
+RUN dnf config-manager \
+        --add-repo \
+        https://download.docker.com/linux/fedora/docker-ce.repo -y && \
+        dnf makecache fast -y && \
+        dnf install \
+        docker-ce \
+        -y  && \
+        dnf clean all
+
+ENV container docker
+#RUN rm /usr/lib/systemd/system/lvm2-lvmetad.service
+RUN curl -s https://pkg.cfssl.org/R1.1/cfssl_linux-amd64 -o /usr/bin/cfssl && \
+    chmod +x /usr/bin/cfssl
+RUN curl -s https://pkg.cfssl.org/R1.1/cfssljson_linux-amd64 -o /usr/bin/cfssljson && \
+    chmod +x /usr/bin/cfssljson
+
+# We need a specific version to work with what's shipping in CoreOS
```

Comment:
As discussed we can't at the moment -I've updated the comment to explain for now.